### PR TITLE
Fix verbose init crash

### LINKED ISSUE #368: Crash when enabling verbose output
Please paste the output of `uzbl-core --bug-info` here:

```
Commit: v0.9.1-163-g4fbf619
GTK compile: 3.22.10
GTK run: 3.22.10
WebKit compile: 2.16.0
WebKit run: 2.16.0
WebKit2: 1
libsoup compile: 2.56.0
libsoup run: 2.56.0
```

Looks like the window isn't valid at the point where we try to dump information about it.

```
#0  0x00007ffff3a0b5e2 in _gdk_window_has_impl () at /lib64/libgdk-3.so.0
#1  0x00007ffff3a41c3e in gdk_x11_window_get_xid () at /lib64/libgdk-3.so.0
#2  0x000000000040cea2 in main (argc=1, argv=0x7fffffffb308) at src/uzbl-core.c:410
```

### PR DESCRIPTION
**Pull request by keis**
Fixes #368

### CODE REVIEW

File: src/commands.c [hunk 1]
```diff
@@ -2632,9 +2632,7 @@ IMPLEMENT_COMMAND (exit)
         gtk_widget_destroy (GTK_WIDGET (uzbl.gui.plug));
     }
 
-    if (uzbl.state.gtk_started) {
-        gtk_main_quit ();
-    }
+    gtk_main_quit ();
```
Comment (mathstuf) [inline src/commands.c]: Does this crash if you put `exit` into the configuration file? GTK isn't started before we read the configuration file.

Comment (mathstuf) [inline src/commands.c]: Oh, your commit message has more context.

**Comment by keis** [comment]
I don't really know if there is something like the XID in the wayland world, or how you would use it.

**Comment by mathstuf** [comment]
Hmm, true.

**Pull request merged**